programmers/실패율.js

```diff
@@ -0,0 +1,19 @@
+const N = 5;
+const stages = [2, 1, 2, 6, 2, 4, 3, 3];
+function solution(N, stages) {
+    const answer = [];
+    const result = [];
+    stages.sort((a, b) => a - b);
+    for (let i = 1; i < N + 1; i++) {
+        const count = stages.filter(element => i === element).length;
+        answer.push(count/ (stages.length - stages.indexOf(i)));
+    }
+    for (let j = 0; j < N; j++) {
+        const maximum = answer.indexOf(Math.max.apply(null, answer));
+        result.push(maximum + 1);
+        answer.splice(maximum, 1, null);
+    }
+    return result;
+}
+
+console.log(solution(N, stages));
```

Comment:
잘푸셨네요 굿굿
컨벤션만 신경써주시면 완벽할거같아요 :)
1, 2, 19 줄은 테스트 위해서 작성해 주신거 같은데 pr 보내실때 그 부분은 지워서 보내는게 깔끔할거같아요!
```suggestion
function solution(N, stages) {
    const answer = [];
    const result = [];
    
    stages.sort((a, b) => a - b);
    
    for (let i = 1; i < N + 1; i++) {
        const count = stages.filter(element => i === element).length;
        answer.push(count / (stages.length - stages.indexOf(i)));
    }
     for (let j = 0; j < N; j++) {
        const maximum = answer.indexOf(Math.max.apply(null, answer));
        result.push(maximum + 1);
        answer.splice(maximum, 1, null);
    }
    return result;
}
```